samples/bluetooth/direction_finding_connectionless_tx/README.rst

```diff
@@ -0,0 +1,154 @@
+.. _direction_finding_connectionless_tx:
+
+Bluetooth: Direction finding connectionless beacon
+##################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+The direction finding connectionless beacon sample demonstrates Bluetooth LE Direction Finding transmission.
+It uses the Constant Tone Extension (CTE), that is transmitted with periodic advertising PDUs.
+
+The sample supports two Direction Finding modes:
+
+* Angle of Arrival
+* Angle of Departure
+
+By default, both modes are available in the sample.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52833dk_nrf52833
+
+The sample also requires an antenna matrix when operating in Angle of Departure mode.
+It can be a Nordic Semiconductor design 12 patch antenna matrix, or any other antenna matrix.
+
+Configuration
+*************
+
+|config|
+
+Angle of Arrival mode
+=====================
+
+To build this sample with Angle of Arrival mode only, set :makevar:`OVERLAY_CONFIG`
+to :file:`overlay-aoa.conf`.
+
+See :ref:`cmake_options` for instructions on how to add this option.
+For more information about using configuration overlay files, see :ref:`zephyr:important-build-vars` in the Zephyr documentation.
+
+Antenna matrix configuration for angle of Departure
+===================================================
+
+To use this sample with Angle of Departure enabled, additional configuration of GPIOs is required to control the antenna array.
+Example of such configuration is provided in devicetree overlay file :file:`nrf52833dk_nrf52833.overlay`.
+
+The overlay file provides provides the information about which GPIOS should be used by Radio peripheral to switch between antenna patches during CTE transmission in AoD mode.
+At least two GPIOs must be provided to enable antenna switching.
+
+The GPIOs are used by the Radio peripheral in order given by the ``dfegpio#-gpios`` properties.
+The order is important because it affects mapping of antenna switch patterns to GPIOS (see `Antenna patterns`_).
```

Comment:
```suggestion
The order is important because it affects mapping of the antenna switching patterns to GPIOs (see `Antenna patterns`_).
```

Reply:
Added 'the', corrected switch->switching, GPIOS->GPIOs